Rubberduck.Core/UI/CodeExplorer/Commands/AddPredeclaredClassModuleCommand.cs

```diff
@@ -0,0 +1,48 @@
+using NLog;
+using Rubberduck.Navigation.CodeExplorer;
+using Rubberduck.UI.Command;
+using Rubberduck.VBEditor.SafeComWrappers;
+using System.IO;
+
+namespace Rubberduck.UI.CodeExplorer.Commands
+{
+    [CodeExplorerCommand]
+    public class AddPredeclaredClassModuleCommand : CommandBase
+    {
+        private readonly AddComponentCommand _addComponentCommand;
+
+        public AddPredeclaredClassModuleCommand(AddComponentCommand addComponentCommand) : base(LogManager.GetCurrentClassLogger())
+        {
+            _addComponentCommand = addComponentCommand;
+        }
+
+        protected override bool EvaluateCanExecute(object parameter)
+        {
+            return _addComponentCommand.CanAddComponent(parameter as CodeExplorerItemViewModel);
+        }
+
+        protected override void OnExecute(object parameter)
+        {
+            string moduleText = CreatePreclaredClassModule();
+            _addComponentCommand.AddComponent(parameter as CodeExplorerItemViewModel, moduleText);
+        }
+
+        private string CreatePreclaredClassModule()
+        {
+            //module text intentionally omits a VB_Name attribute so that the name is automatically assigned by the VBE.
+            string moduleText = @"
```

Comment:
Why hard-code the template here? I would rather that we create a new resource file and stuff that template. That would make it easier to customize via settings in a future PR.